src/mod_muc.erl

```diff
@@ -607,7 +606,7 @@ get_rooms(ServerHost, Host) ->
     Mod = gen_mod:db_mod(LServer, ?MODULE),
     Mod:get_rooms(LServer, Host).
 
-load_permanent_rooms(Host, ServerHost, Access,
+load_all_rooms(Host, ServerHost, Access,
```

Comment:
why rename this?

Reply:
Because we are now loading temporary rooms too.

Reply:
Oh because we're not clearing from the archive table, it loads all the room history automatically?

Reply:
Oh hmmm I think this call should be removed, actually

Reply:
Removed.  Now they're not loaded into memory upon initialization.

Reply:
why?  Also isn't this part of #4 (in #feature-chat)?  I thought we were doing a separate PR for that.